composer.json

```diff
@@ -22,28 +22,28 @@
     }
   ],
   "require": {
-    "php": ">=7.0",
+    "php": "^7.2",
     "ocramius/proxy-manager": "2.*",
     "psr/log": "1.*",
     "psr/cache": "1.0.*",
     "doctrine/annotations": "~1.3",
     "psr/container": "^1.0",
-    "symfony/dependency-injection": ">3.3"
+    "symfony/dependency-injection": "^5.0"
   },
   "suggest": {
-    "symfony/cache": "3.*"
+    "symfony/cache": "^5.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "5.*",
+    "phpunit/phpunit": "^7.0",
     "squizlabs/php_codesniffer": "2.*",
     "phpmd/phpmd" : "@stable",
     "phing/phing": "@stable",
     "sebastian/phpcpd": "~2",
-    "symfony/cache": "3.*",
+    "symfony/cache": "^5.0",
```

Comment:
Are you sure that this restriction will work on Symfony 4? You added:
`- SYMFONY_VERSION=4.4.*`

Reply:
Symfony 4.4 was removed